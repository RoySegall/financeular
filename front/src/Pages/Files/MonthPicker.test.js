describe('Month picker', () => {

  it('Testing the months are displayed correctly and the switching with months', () => {
    expect(1).toBe(2);
  })

});