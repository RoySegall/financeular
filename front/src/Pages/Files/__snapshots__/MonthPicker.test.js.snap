// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Month picker Testing the component months with a picked month 1`] = `
<ul
  className="flex mt-auto w-full border border-yellow-600 mb-2 bg-white rounded"
>
  <li
    className="p-2 bg-green-dark text-light-white"
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      January 2019
    </a>
  </li>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      February 2019
    </a>
  </li>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      March 2019
    </a>
  </li>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      January 2020
    </a>
  </li>
</ul>
`;

exports[`Month picker Testing the component months without a picked month 1`] = `
<ul
  className="flex mt-auto w-full border border-yellow-600 mb-2 bg-white rounded"
>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      January 2019
    </a>
  </li>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      February 2019
    </a>
  </li>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      March 2019
    </a>
  </li>
  <li
    className="p-2 "
  >
    <a
      className="cursor-pointer"
      onClick={[Function]}
    >
      January 2020
    </a>
  </li>
</ul>
`;
